reference/shared/document.getselecteddataasync.md

```diff
@@ -15,21 +15,46 @@ Office.context.document.getSelectedDataAsync(coercionType [, options], callback)
 
 ## Parameters
 
+| Name           | Type               | Description |
+|:---------------|:-------------------|:------------|
+| _coercionType_ | [CoercionType](#coerciontype)    | The type of data structure to return. Required.
+| _options_      | [object](#options) | Specifies any of the following [optional parameters](../../docs/develop/asynchronous-programming-in-office-add-ins.md#passing-optional-parameters-to-asynchronous-methods)
+| _callback_     | [function](#callback-value) | A function that is invoked when the callback returns, whose only parameter is of type  [AsyncResult][].
```

Comment:
Please provide a URL for the **AsyncResult** link at the end of this line.

Reply:
See command bellow